cirq-core/cirq/ops/common_channels.py

```diff
@@ -717,33 +717,33 @@ def _qasm_(self, args: 'cirq.QasmArgs', qubits: Tuple['cirq.Qid', ...]) -> Optio
     def _qid_shape_(self):
         return (self._dimension,)
 
-    def _act_on_(self, args: 'cirq.ActOnArgs', qubits: Sequence['cirq.Qid']):
-        from cirq import sim, ops
+    def _act_on_(self, args: 'cirq.OperationTarget', qubits: Sequence['cirq.Qid']):
+        if len(qubits) != 1:
+            return NotImplemented
 
-        if isinstance(args, sim.ActOnStabilizerCHFormArgs):
-            axe = args.qubit_map[qubits[0]]
-            if args.state._measure(axe, args.prng):
-                ops.X._act_on_(args, qubits)
-            return True
+        class PlusGate(raw_types.Gate):
```

Comment:
[No change needed] Adding a new internal gate class just for this interaction feels heavy-handed, but I guess it's necessary so we can use `act_on` with the other simulator types?

Reply:
I was just thinking, maybe it makes sense to have a public PlusGate? This code is copied and pasted in several places, mostly tests.

Reply:
Just saw the test from your other comment. I wouldn't be opposed to a public PlusGate (there might even be hardware examples of such a gate), but it can be added in a separate PR.